osu.Game/Screens/Play/Player.cs

```diff
@@ -165,16 +180,20 @@ private void load(AudioManager audio, BeatmapDatabase beatmaps, OsuConfigManager
 
             Children = new Drawable[]
             {
-                new Container
+                hitRendererContainer = new Container
                 {
                     RelativeSizeAxes = Axes.Both,
-                    Clock = interpolatedSourceClock,
                     Children = new Drawable[]
                     {
-                        HitRenderer,
-                        skipButton = new SkipButton
+                        new Container
                         {
-                            Alpha = 0
+                            RelativeSizeAxes = Axes.Both,
+                            Clock = offsetClock,
+                            Children = new Drawable[]
+                            {
+                                HitRenderer,
+                                skipButton = new SkipButton { Alpha = 0 },
+                            }
                         },
                     }
                 },

```

Comment:
Is there a reason we have two nested `Containers` here? Can't this be `hitRendererContainer { Clock = offsetClock, Children = { hitRenderer, skipButton } }` ?